docs/windows.md

```diff
@@ -348,69 +256,49 @@ The script `Prepare-AntreaAgent.ps1` performs following tasks:
-time you restart the Node to prepare the environment for antrea-agent.
+The script must be executed every time you restart the Node to prepare the
+environment for antrea-agent.
 
-You could make the script be executed automatically after Windows startup by
-using different methods. Here're two examples for your reference:
+You can ensure that the script is executed automatically after each Windows
+startup by using different methods. Here are two examples for your reference:
 
-* Example1: Update kubelet service.
+* Example 1: Update kubelet service.
 
 Insert following line in kubelet service script `c:\k\StartKubelet.ps1` to invoke
 `Prepare-AntreaAgent.ps1` when starting kubelet service:
 
 ```powershell
-& C:\k\antrea\Prepare-AntreaAgent.ps1
+& C:\k\antrea\Prepare-AntreaAgent.ps1 -InstallKubeProxy $false -RunOVSServices $false
 ```
 
-* Example2: Create a ScheduledJob that runs at startup.
+* Example 2: Create a ScheduledJob that runs at startup.
 
 ```powershell
 $trigger = New-JobTrigger -AtStartup -RandomDelay 00:00:30 
 $options = New-ScheduledJobOption -RunElevated
-Register-ScheduledJob -Name PrepareAntreaAgent -Trigger $trigger  -ScriptBlock { Invoke-Expression C:\k\antrea\Prepare-AntreaAgent.ps1 } -ScheduledJobOption $options
+Register-ScheduledJob -Name PrepareAntreaAgent -Trigger $trigger  -ScriptBlock { Invoke-Expression C:\k\antrea\Prepare-AntreaAgent.ps1 -InstallKubeProxy $false -RunOVSServices $false } -ScheduledJobOption $options
 ```
 
 ##### 5. Run kubeadm to join the Node
 
-On Windows Node, run the `kubeadm join` command to join the cluster. The token
-is provided by the control-plane Node. If you forgot the token, or the token
-has expired, you can run `kubeadm token create --print-join-command`
-(on the control-plane Node) to generate a new token and join command. An example
+On Windows Nodes, run the `kubeadm join` command to join the cluster. The token
+is provided by the control-plane Node. If you lost the token, or the token has
+expired, you can run `kubeadm token create --print-join-command` (on the
+control-plane Node) to generate a new token and join command. An example
 `kubeadm join` command is like below:
 
 ```powershell
 kubeadm join 192.168.101.5:6443 --token tdp0jt.rshv3uobkuoobb4v  --discovery-token-ca-cert-hash sha256:84a163e57bf470f18565e44eaa2a657bed4da9748b441e9643ac856a274a30b9
 ```
 
-Then, set the Node IP used by kubelet.
-Open file `/var/lib/kubelet/kubeadm-flags.env`:
-
-```text
-KUBELET_KUBEADM_ARGS="--network-plugin=cni --pod-infra-container-image=mcr.microsoft.com/oss/kubernetes/pause:1.3.0"
-```
-
-Append `--node-ip=$NODE_IP` at the end of params. Replace `$NODE_IP` with
-the address for kubelet. It should look like:
-
-```text
-KUBELET_KUBEADM_ARGS="--network-plugin=cni --pod-infra-container-image=mcr.microsoft.com/oss/kubernetes/pause:1.3.0 --node-ip=$NODE_IP"
-```
-
-Restart kubelet service for changes to take effect.
-
-```powershell
-restart-service kubelet
-```
```

Comment:
This no longer seems to be required: the `--node-ip` command-line flag is already included in the generated `c:\k\StartKubelet.ps1`.

```powershell
PS C:\Users\Administrator> cat c:\k\StartKubelet.ps1
$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
$global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")

$global:KubeletArgs += " --cert-dir=$env:SYSTEMDRIVE\var\lib\kubelet\pki --config=/var/lib/kubelet/config.yaml --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"mcr.microsoft.com/oss/kubernetes/pause:1.4.1`" --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`" --resolv-conf=`"`" --node-ip=$env:NODE_IP"
$cmd = "C:\k\kubelet.exe $global:KubeletArgs"
Invoke-Expression $cmd
```